src/test/java/com/example/springTask/MapperTests.java

```diff
@@ -0,0 +1,52 @@
+package com.example.springTask;
+
+import com.example.springTask.dto.StudentDTO;
+import com.example.springTask.dto.TeacherDTO;
+import com.example.springTask.mappers.StudentMapperImpl;
+import com.example.springTask.mappers.TeacherMapperImpl;
+import com.example.springTask.models.Student;
+import com.example.springTask.models.Teacher;
+import org.junit.Before;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+
+@SpringBootTest
+public class MapperTests {
+    @Autowired
+    private TeacherMapperImpl teacherMapper;
+
+    @Autowired
+    private StudentMapperImpl studentMapper;
+
+    @Before
+    public void setup() {
+        teacherMapper = new TeacherMapperImpl();
```

Comment:
no need to initialize it. 


<!--StartFragment-->    @Autowired    private TeacherMapperteacherMapper;<!--EndFragment-->
must be enouth.